source/develop/release-management/features/virt/headless-vm.html.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Headless VM 
+category: feature
+authors: SharonG
+wiki_category: Feature
+wiki_title: Features/Headless VM
```

Comment:
You don't need the 'wiki' tags. They are for the pages that were imported from the old wiki.
It is the reason why the CI is failing on this commit.
Please remove 'wiki_title' and 'wiki_category'